libs/blocks/merch/merch.js

```diff
@@ -4,7 +4,7 @@ import {
 import { replaceKey } from '../../features/placeholders.js';
 
 export const PRICE_LITERALS_URL = 'https://milo.adobe.com/libs/commerce/price-literals.json';
-export const CHECKOUT_LINK_CONFIG_PATH = '/commerce/checkout-link.json'; // relative to libs.
+export const CHECKOUT_LINK_URL = 'https://www.adobe.com/federal/commerce/checkout-link.json'; // relative to libs.
```

Comment:
as GWP, how do I validate my change before publishing it live?
should we use https://www.stage.adobe.com/federal/commerce/checkout-link.json?
Does it point to hlx.page content?
cc @mokimo

Reply:
no, stage.adobe.com is not yet connected to stage/federal/hlx.page. I'd recommend to use stage--federal--adobecom.hlx.page/commerce/checkout-link.json
https://github.com/orgs/adobecom/discussions/2220#discussioncomment-9260878 

until we properly set up stage as part of the helix-5 migration